packages/babel-helpers/src/helpers.js

```diff
@@ -1043,8 +1043,24 @@ helpers.createForOfIteratorHelper = helper("7.9.0")`
   // f: finish (always called at the end)
 
   export default function _createForOfIteratorHelper(o) {
-    if (typeof Symbol === "undefined" || o[Symbol.iterator] == null)
+    if (typeof Symbol === "undefined" || o[Symbol.iterator] == null) {
+      // Fallback for engines without symbol support
+      if (Array.isArray(o)) {
+        var i = 0;
+        var F = function(){};
+        return {
+          s: F,
+          n: function() {
+            if (i >= o.length) return { done: true };
+            return { done: false, value: o[i++] };
+          },
+          e: F,
```

Comment:
This `e` method needs to throw the passed in error.

Reply:
Thanks, good catch!